app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -121,6 +121,12 @@
 
           ga('create', 'UA-26179049-6', '<%= ENV.fetch('GOVUK_APP_DOMAIN') %>');
           ga('set', 'anonymizeIp', true);
+
+          <% if content_for?(:custom_dimension) %>
+            [index, value] = JSON.parse('<%= content_for(:custom_dimension) %>');
```

Comment:
Could the value of `:custom_dimension` simply be the JS to call?
Maybe: `:custom_dimension_js` which would have the value `GOVUKAdmin.setDimension(4, 'something');`